scipy/stats/_stats_py.py

```diff
@@ -1817,20 +1817,97 @@ def jarque_bera(x, *, axis=None):
+    skewness and kurtosis.
+
     >>> from scipy import stats
-    >>> rng = np.random.default_rng()
-    >>> x = rng.normal(0, 1, 100000)
-    >>> jarque_bera_test = stats.jarque_bera(x)
-    >>> jarque_bera_test
-    Jarque_beraResult(statistic=3.3415184718131554, pvalue=0.18810419594996775)
-    >>> jarque_bera_test.statistic
-    3.3415184718131554
-    >>> jarque_bera_test.pvalue
-    0.18810419594996775
+    >>> res = stats.jarque_bera(x)
+    >>> res.statistic
+    6.982848237344646
+    >>> s = stats.skew(x)  # calculate the sample skewness
+    >>> k = stats.kurtosis(x)  # calculate the sample kurtosis
+    >>> len(x)/6 * (s**2 + k**2/4)  # calculate the Jarque-Bera statistic
+    6.982848237344646
+
+    Because the normal distribution has zero skewness and zero
+    ("excess" or "Fisher") kurtosis, the value of this statistic tends to be
+    low for samples drawn from a normal distribution.
+
+    The test is performed by comparing the observed value of the statistic
+    against the null distribution: the distribution of statistic values derived
+    under the null hypothesis that the sample *is* normally distributed.
+    For the Jarque-Bera test, the null distribution for very large samples is
+    the chi-squared distribution with two degrees of freedom.
+
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots()
+    >>> dist = stats.chi2(df=2)
+    >>> jb_val = np.linspace(0, 10, 100)
+    >>> ax.plot(jb_val, dist.pdf(jb_val))
+    >>> ax.set_title("PDF of Jarque-Bera Null Distribution")
+    >>> ax.set_xlabel("statistic")
+    >>> ax.set_ylabel("probability density")
+    >>> plt.show()
+
+    The comparison is quantified by the p-value: the proportion of values in
+    the null distribution greater than or equal to the observed value of the
+    statistic.
+
+    >>> dist.sf(res.statistic)
+    0.03045746622458189
+    >>> res.pvalue
+    0.03045746622458189
+
+    If the p-value is small - that is, if there is a low probability of
+    sampling data from a normally distributed population that produces such an
+    extreme value of the statistic - this may be taken as evidence against
+    the null hypothesis in favor of the alternative: the sample was not
+    drawn from a normal distribution. The inverse is not true; that is,
+    the test is not used to provide evidence for the null hypothesis.
+
+    Note that the chi-squared distribution provides an asymptotic approximation
+    of the null distribution; it is only accurate for samples with many
+    observations. For small samples like ours, `scipy.stats.monte_carlo_test`
+    may provide a more accurate, albeit stochastic, approximation of the
+    exact p-value.
+
+    >>> def statistic(x, axis):
+    ...     s = stats.skew(x, axis=axis)
+    ...     k = stats.kurtosis(x, axis=axis)
+    ...     return x.shape[axis]/6 * (s**2 + k**2/4)
+    >>> res = stats.monte_carlo_test(x, stats.norm.rvs, statistic)
+    >>> fig, ax = plt.subplots()
```

Comment:
We need to adjust a big so the legend does not overlap
```suggestion
    >>> fig, ax = plt.subplots(figsize=(8, 5))
```